auto.config.ts

```diff
@@ -0,0 +1,18 @@
+import { AutoRc } from "auto";
+
+
+/** Auto configuration */
+export default function rc(): AutoRc {
+    return {
+        plugins: [
+            "npm",
+            "conventional-commits",
+            "first-time-contributor",
```

Comment:
What does this do?

Reply:
When someone contributes to a repository for the first time, it tags and thanks them in the first [Changelog heading](https://github.com/nteract/data-explorer/blob/main/CHANGELOG.md#v8210-tue-jun-29-2021) and [Release Notes](https://github.com/nteract/data-explorer/releases/tag/v8.2.10) , and doesn't do anything if this release doesn't have any 1st time contributors. 

Docs: https://intuit.github.io/auto/docs/generated/first-time-contributor

We can remove this if it wouldn't be relevant for the Vega projects, but I figured it couldn't hurt + might help with encouraging people to become ongoing contributors.